### PR TITLE
leaflet.esri.renderers

### PR DESCRIPTION
**Pull request by jgravois**
thanks!

### CODE REVIEW

**Comment by jgravois** [comment]
i'll drop `.git` from the url

File: files/leaflet.esri.renderers/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Rachel Nehmer"
+github = "https://github.com/Esri/esri-leaflet-related.git"
```
Comment (megawac) [inline files/leaflet.esri.renderers/info.ini]: renderers not related

Comment (jgravois) [inline files/leaflet.esri.renderers/info.ini]: fixed and rebased.  thanks!

someday i'll submit a PR here that doesn't have an immediately obvious problem :smile:

Comment (megawac) [inline files/leaflet.esri.renderers/info.ini]: :D heh, I used to mess my submits all the time before I wrote a quick setup script https://github.com/megawac/cdn-it

**Pull request merged**